packages/cli/docs-resolver/src/DocsDefinitionResolver.ts

```diff
@@ -785,7 +808,7 @@ export class DocsDefinitionResolver {
             orphaned: item.orphaned,
             pageId,
             authed: undefined,
-            noindex: undefined,
+            noindex: item.noindex ?? this.markdownFilesToNoIndex.get(item.absolutePath),
```

Comment:
same comment as above -- should this be landingPageConfig.noindex || this.markdownFilesToNoIndex.get(landingPageConfig.absolutePath)